src/components/Notification/Notification.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import { Toast, ToastHeader, ToastBody, Button } from 'reactstrap'
+import Icon from '../Icon/Icon'
+import classNames from 'classnames'
+import PropTypes from 'prop-types'
+
+const propTypes = {
+  header: PropTypes.string,
+  content: PropTypes.string,
+  state: PropTypes.string,
```

Comment:
Probably `status` instead of `state` IMHO. What do you think?

Reply:
It could have been `status` but it has been given as `state` in BI. So, I believe we should stick to that ?

Reply:
If it was `state` already nevermind, keep it that way.